container_files/Containerfile.services

```diff
@@ -22,7 +22,7 @@ RUN TAG=$tag cargo build -p trust --release
 
 FROM registry.access.redhat.com/ubi9/ubi:latest as frontendbuilder
 
-ARG RUST_VERSION="1.74.0"
```

Comment:
Same as the next comment. I just want to double-check downgrading the Rust version is really needed